docs/frameworks/index.md

```diff
@@ -30,6 +30,7 @@ You can find all the `vite-plugin-pwa` virtual modules declarations in the follo
 declare module 'virtual:pwa-register' {
   export interface RegisterSWOptions {
     immediate?: boolean
+    onBeginUpdate?: () => void
```

Comment:
remove docs changes, the docs folder here is deprecated (we need to add it to the docs repo): we have a redirection to new docs website

Reply:
Copy